### ISSUE TITLE
Blocking user I have no chat with

### ISSUE THREAD

**Issue by zevlg** [tdlib/td#1252]
Recently TDLib API moved "is_blocked" property from user to the chat.  And provides `toggleChatIsBlocked` method to mark chats as blocked.

Sometimes I want to block users I have no chat with, so he won't be able to initiate private chat with me in the future.  How this can be done using new API?

Thanks

**Comment by levlam** [comment]
You need to call `createPrivateChat` with `force` == true and then pass ID of the returned chat.

But this API probably needs to be reworked to remove this step.

**Comment by zevlg** [comment]
> But this API probably needs to be reworked to remove this step.

I’ll keep this issue open then

**Comment by levlam** [comment]
The next TDLib version has
```
messageSenderUser user_id:int32 = MessageSender;
messageSenderChat chat_id:int53 = MessageSender;
```
to represent a sender of a message. So it is possible to replace `toggleChatIsBlocked` with `toggleMessageSenderIsBlocked`, `blockChatFromReplies` with `blockMessageSencerFromReplies`, `getBlockedChats` with `getBlockedMessageSenders` and return the field `is_blocked` to `userFullInfo`.

**Comment by levlam** [comment]
This was implemented in TDLib 1.6.10.
